Networking/Networking/Model/Order.swift

```diff
@@ -221,8 +221,8 @@ public struct Order: Decodable, Sendable, GeneratedCopiable, GeneratedFakeable {
         // https://github.com/woocommerce/woocommerce/blob/3611d4643791bad87a0d3e6e73e031bb80447417/plugins/woocommerce/includes/class-wc-order.php#L1537-L1561
         let needsProcessing = try container.decodeIfPresent(Bool.self, forKey: .needsProcessing) ?? false
 
-        // Filter out metadata if the key is prefixed with an underscore (internal meta keys) or the value is empty
-        let customFields = allOrderMetaData?.filter({ !$0.key.hasPrefix("_") && !$0.value.isEmpty }) ?? []
+        // Filter out metadata if the key is prefixed with an underscore (internal meta keys)
+        let customFields = allOrderMetaData?.filter({ !$0.key.hasPrefix("_") }) ?? []
```

Comment:
While testing, I noticed that some fields were "lost" after saving then going to the order details, and it turns out we were stripping the fields with empty values here, since custom fields can have empty values, and we allow it also in the app, I updated this part, with this, the logic here is also aligned with the other places ([MetaDataMapper](https://github.com/woocommerce/woocommerce-ios/blob/98f82341911f06985ec6d58b2a84ddc579924539/Networking/Networking/Mapper/MetaDataMapper.swift#L21) and [Product](https://github.com/woocommerce/woocommerce-ios/blob/98f82341911f06985ec6d58b2a84ddc579924539/Networking/Networking/Model/Product/Product.swift#L529)).

Reply:
Thanks, can confirm that it's the right behavior and we want to allow custom fields with empty values.